util/appconfig/load.go

```diff
@@ -1,16 +1,15 @@
-package main
+package appconfig
 
 import (
 	"io"
 	"os"
-	"strconv"
-	"strings"
 
 	"github.com/moby/buildkit/cmd/buildkitd/config"
```

Comment:
This doesn't really solve the `util` issue if `util` is importing pkg from `cmd/buildkitd`.